<launch>
    <!-- Tracker node -->
    <node pkg="blort_ros" type="gltracker_node" name="blort_tracker" args="$(find blort_ros)" output="screen">
        <param name="launch_mode" value="tracking" />
        <remap from="/detector_camera_info" to="/camera/camera_info" />
        <remap from="/detector_image" to="/histogram_segmentation/image_masked" />
        <remap from="/tracker_image" to="/camera/image_raw" />           
    </node>
    
    <!-- Detector node -->
    <node pkg="blort_ros" type="gldetector_node" name="blort_detector" args="$(find blort_ros)">
        <remap from="/blort_camera_info" to="/camera/camera_info" />
    </node>
</launch>

